scripts/publish_posts.py

```
@@ -120,13 +376,40 @@ def publish_post_if_ready(file_path: Path):
+            if seo_updated or images_updated > 0:
+                # Update YAML with enhanced metadata
+                new_yaml = original_yaml
+                if seo_updated:
+                    new_yaml = update_yaml_frontmatter(original_yaml, enhanced_metadata)
+
+                try:
+                    new_content = f"---\n{new_yaml.strip()}\n---\n{updated_body}"
+                    file_path.write_text(new_content, encoding='utf-8')
+                    logging.info(f"Successfully enhanced {file_name}")
+                    if seo_updated:
+                        logging.info(f"  - Enhanced SEO metadata")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)